docs/howto/kubernetes/gardener/hibernate-shoot-cluster.md

```diff
@@ -0,0 +1,74 @@
+---
+description: How to hibernate a Gardener-based Kubernetes cluster
+---
+# Hibernating a Kubernetes cluster
+
+There will be times when you won't be using your Gardener-based cluster
+much, if at all. To save on costs, you can put the whole cluster in
+hibernation. If you do, then from that time on (and until you wake the
+cluster again), you will not be paying for any CPU and RAM utilization,
+but *only* for the total storage space your cluster occupies. Let us
+see how you can hibernate a cluster from the {{gui}}, and then how you
+can wake up an already hibernated cluster.
+
+## Assumptions
+
+We assume you have already used Gardener in {{brand}} to spin up a
+Kubernetes cluster, which is now humming away. If you've never done
+this before, please feel free to [follow this
+guide](../create-shoot-cluster).
+
+## Hibernating a cluster
+
+Fire up your favorite web browser and navigate to
+<https://{{gui_domain}}>. Make sure the vertical pane at the left-hand
+side of the page is in full view, then choose
+*Containers → [{{k8s_management_service}}](https://{{gui_domain}}/containers/gardener)*.
+In the main pane, you will see your Gardener cluster.  Click anywhere
+on the corresponding row for a detailed view of the various cluster
+characteristics. Go to the *Status* tab and check if hibernation is
+possible.
+
+![Gardener cluster status](assets/garhiber-01.png)
+
+If hibernation is indeed possible, this will be reflected in the
+*Constraints* section. To go ahead and actually hibernate the cluster,
+click the orange three-dot icon at the right-hand side of the cluster
+row, and from the pop-up menu that appears, select *Hibernate Cluster*.
+
+![Hibernate cluster](assets/garhiber-02.png)
+
+A big pop-up window will appear, asking you to confirm the action.
+Click the red button labeled *Yes, Hibernate*.
+
+![Confirm cluster hibernation](assets/garhiber-03.png)
+
+After a minute or two, the cluster will be in a hibernated state. This
+fact will be indicated by the white-square-in-red-circle icon, at the
+left-hand side of the cluster row.
+
+![Cluster in hibernation](assets/garhiber-04.png)
+
+At this point, and as long as the cluster is in hibernation, any
+attempt to list the cluster nodes, e.g., via `kubectl`, will fail.
```

Comment:
Just a micro-suggestion: maybe we should be a little more explicit about the kind of failure that users should expect?

```suggestion
attempt to list the cluster nodes, e.g., via `kubectl`, will time out.
```

... or perhaps

```suggestion
attempt to list the cluster nodes, e.g., via `kubectl`, will fail with a timeout.
```

What do you think?

Reply:
Well, while a command like `kubectl get nodes` does fail, you do get an error message **almost instantaneously**:

`Unable to connect to the server: tls: failed to verify certificate: x509: certificate is valid for ingress.local, not api.ghar.p43597.k8s.cleura.cloud`

That doesn't seem like a timeout error to me, and I'm guessing there's no actual timeout because a load balancer is still in place **even** for hibernated clusters.

So, because of that, I think we could leave the sentence as is.